README.md

```diff
@@ -167,6 +167,17 @@ namespace :deploy do
       end
     end
   end
+
+  # Workaround to map composer and drush commands
```

Comment:
@gido @yannlugrin Can you tell mit if that work based on ttps://github.com/capistrano/composer/issues/22

Reply:
I don't know if this is better than this:
https://github.com/antistatique/watchdreamer/blob/dev/config/deploy/production.rb#L13-L14

Reply:
it did not works, I revert this change